modules/annotationGet.nf

```diff
@@ -2,7 +2,7 @@
 * DOWNLOAD ANNOTATION
 ********************************************/
 process annotationGet {
-    label 'python3'
+    label 'basic_tools'
```

Comment:
we need to check if this clashes with containers... because having two labels confuses the container profile... but I think this could be fine because the basic_tools label is only available in the conda profile, right?

Reply:
Before, container and execution were not separated. 
Now, the logic is: container=basic_tool=wget and execution=smallTask=cpus=1

So, in `nodes.config` and `loca.config` we need `smallTask` (and no `python3` or `basic_tools`), in `conda.config` we need `basic_tools` and in `nodes.config` we need a container with docker. Maybe we just reuse the python3 caontainer?

labels are then merged: https://www.nextflow.io/docs/latest/config.html#selectors-priority

Reply:
Ahh, okay. Yeah, the py3 container should be fine! My only point was that we can run into problems when we define two different containers via two labels because then, I think, nextflow just decides for the container that was defined by the second label. 

Here, I had problems because of constructions like this: 

```
label 'dammit'
label 'smallTask'
``` 

when both define a `container:` in `nodes.config` 

If this is not the case, this should be fine